integration-tests/kyuubi-hive-it/src/test/scala/org/apache/kyuubi/it/hive/operation/KyuubiOperationHiveEnginePerUserSuite.scala

```diff
@@ -61,6 +64,44 @@ class KyuubiOperationHiveEnginePerUserSuite extends WithKyuubiServer with HiveEn
     }
   }
 
+  test("[KYUUBI #5865] Hive engine CLI_ODBC_KEYWORDS") {
+    withSessionConf(Map(KyuubiConf.SERVER_INFO_PROVIDER.key -> "ENGINE"))()() {
+      withSessionHandle { (client, handle) =>
+        val req = new TGetInfoReq()
+        req.setSessionHandle(handle)
+        req.setInfoType(TGetInfoType.CLI_ODBC_KEYWORDS)
+        val value = client.GetInfo(req).getInfoValue.getStringValue
+        if (SemanticVersion(HiveVersionInfo.getVersion).isVersionAtLeast("3.0.0")) {
+          assert(value ===
+            "ABORT,ACTIVATE,ACTIVE,ADMIN,AFTER,ALLOC_FRACTION,ANALYZE,APPLICATION," +
```

Comment:
the list is quite strict and it is likely different in different versions or vendor implementations, simply check it contains some MUST present keywords, like `DATABASE`, `SHOW`